### ISSUE TITLE
Upgrade service, misleading wording

### ISSUE THREAD

**Issue by Leen15** [rancher/rancher#19579]
**What kind of request is this (question/bug/enhancement/feature request):**
enhancement

**Steps to reproduce (least amount of steps as possible):**
Open workloads page, press Edit on one of them.

**Result:**
The title is "Upgrade Service"

**Other details that may be helpful:**
I'm in a workloads list, I press "edit" on one of it, why the title is "Upgrade Service"? What is a "service"?
I press an "Edit" CTA than bring me on a "Upgrade" page and not a "Edit" page.
I expected a title like "Edit Workload", and a "Save" button on the bottom.

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
Rancher 2.2.1
- Installation option (single install/HA):
HA

**Comment by davidnuzik** [comment]
Version: Master (4/23/19)

Text was fixed for the following:

Edit Workload, Edit Container, Edit Global Service and Edit Sidecar. They used to be "Upgrade" for each.

I reviewed the commit and confirmed the correct text is in the correct places. I.e. for when we are in workload details page and pick edit it will mention as subheader "Select container to edit" instead of "Select service launch config to upgrade"
